core/node/libp2p/transport.go

```diff
@@ -64,6 +66,10 @@ func Transports(tptConfig config.Transports) interface{} {
 			opts.Opts = append(opts.Opts, libp2p.Transport(webrtc.New))
 		}
 
+		if tcpEnabled || wsEnabled {
```

Comment:
:thought_balloon:  maybe  enable it only when both are enabled?
This will enable sharing by defalt, but allow users to fix by disabling WS if there are issues with this.

```suggestion
		if tcpEnabled && wsEnabled {
```